app/code/Magento/Authorization/Command/AdminRolesListCommand.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Authorization\Command;
+
+use Magento\Authorization\Model\GetAdminRolesList;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Helper\Table;
+
+/**
+ * Class AdminRolesListCommand is cli command for showing all existing admin role ids and names
+ */
+class AdminRolesListCommand extends Command
+{
+    /**
+     * @var GetAdminRolesList
+     */
+    private $getAdminRolesListService;
+
+    /**
+     * Constructor
+     *
```

Comment:
```suggestion
```

Reply:
This was added because of static tests...

Reply:
In another file you don't have it and it works well